src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.component.spec.js

```diff
@@ -329,6 +343,49 @@ describe('checkout', () => {
           expect(self.controller.onPaymentChange).toHaveBeenCalledWith({ selectedPaymentMethod: undefined })
           expect(self.controller.orderService.storeCoverFeeDecision).not.toHaveBeenCalled()
         })
+
+        it('should reset securityCode viewValue on switch payment', () => {
```

Comment:
```suggestion
        it('should reset securityCode viewValue', () => {
```

`on switch payment` is implied based on where it is in the code.